lib/uplinkc/uplink_definitions.h

```diff
@@ -36,8 +36,11 @@ typedef struct UplinkConfig {
     struct {
         struct {
             bool skip_peer_ca_whitelist;
+            char *peer_ca_whitelist_path;
         } tls;
         // TODO: add support for MaxMemory
+        char *peer_id_version;
+        char *partner_id;
```

Comment:
`partner_id` is currently intentionally missing, because the semantics how it works needs to change.

Reply:
Ah, didn't know that. Just wanted to be on-pair with libuplink. Shall I remove it? What is planned here?

Reply:
We plan to switch to user agent strings instead of using uuid-s. Which, will be more friendly to supply versions etc. different information.

Reply:
Is it in the near future so I should wait instead of adding it now and removing it again later?

Reply:
Yeah, I'm currently working on making all the changes to support it throughout the codebase. I'll add it once it can be used.

Reply:
So I'll remove the field from this PR again and wait for your additions.